src/payment/checkout/payment-form/CardDetails.jsx

```diff
@@ -95,6 +98,7 @@ export class CardDetailsComponent extends React.Component {
 
   render() {
     const { disabled, isPaymentVisualExperiment } = this.props;
+    const loading = this.props.captureKeyId === '';
```

Comment:
Is this variable being used?  Doesn't seem to be.

Reply:
heh, yup, that's dead code from an earlier version